src/AspNet.Security.OAuth.AdobeIO/AdobeIOAuthenticationDefaults.cs

```diff
@@ -0,0 +1,64 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+
+namespace AspNet.Security.OAuth.AdobeIO
+{
+    public static class AdobeIOAuthenticationDefaults
+    {
+        /// <summary>
+        /// Default value for <see cref="AuthenticationScheme.Name"/>.
+        /// </summary>
+        public const string AuthenticationScheme = "AdobeIO";
+
+        /// <summary>
+        /// Default value for <see cref="AuthenticationScheme.DisplayName"/>.
+        /// </summary>
+        public const string DisplayName = "AdobeIO";
+
+        /// <summary>
+        /// Default value for <see cref="AuthenticationSchemeOptions.ClaimsIssuer"/>.
+        /// </summary>
+        public const string Issuer = "AdobeIO";
+
+        /// <summary>
+        /// Default value for <see cref="RemoteAuthenticationOptions.CallbackPath"/>.
+        /// </summary>
+        public const string CallbackPath = "/signin-adobeio";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.AuthorizationEndpoint"/>.
+        /// </summary>
+        public const string AuthorizationEndpoint = "https://ims-na1.adobelogin.com/ims" + AuthorizationEndpointPath;
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.TokenEndpoint"/>.
+        /// </summary>
+        public const string TokenEndpoint = "https://ims-na1.adobelogin.com/ims" + TokenEndpointPath;
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.UserInformationEndpoint"/>.
+        /// </summary>
+        public const string UserInformationEndpoint = "https://ims-na1.adobelogin.com/ims" + UserInformationEndpointPath;
```

Comment:
Preferred to just use literals, rather than concat with other constants. It makes it easier to avoid bugs caused by missing slashes or double slashes.

Reply:
I was going by example of what the GitHub provider did, but sure.

Reply:
The GitHub provider is slightly different because it supports custom domains to support GitHub Enterprise. The Okta provider and some others have a similar pattern.

Reply:
You can also remove the three `*Path` constants from this file, just the `*Endpoint` ones should be sufficient.

Reply:
Done.